packages/relay-runtime/store/RelayModernEnvironment.js

```diff
@@ -192,6 +192,10 @@ class RelayModernEnvironment implements Environment {
     let optimisticResponse;
     return this._network
       .execute(operation.node, operation.variables, cacheConfig || {})
+      .map(executePayload => {
+        normalizePayload(executePayload);
```

Comment:
won't this cause 2 normalizePayload on `executePayload`?

Reply:
Yep, that's true.

The first one is being used as validation and is not actually returned, so in the case that we need the value normalized, we have to run it again.  There would certainly be more drastic refactors possible, if anyone wants to suggest something.

I was just aiming for a minimal, low-risk change that definitely fixed the error.